bands/models.py

```diff
@@ -103,33 +133,41 @@ def uploaded_stage_plot(self):
         return os.path.exists(self.stage_plot_fpath)
 
     @property
-    def w9_extension(self):
-        return extension(self.w9_filename)
-
-    @property
-    def bio_pic_extension(self):
-        return extension(self.bio_pic_filename)
-
-    @property
     def stage_plot_extension(self):
         return extension(self.stage_plot_filename)
 
     @property
-    def all_badges_claimed(self):
-        return not any(a.is_unassigned for a in self.group.attendees)
+    def completed(self):
+        return self.uploaded_stage_plot
 
-    @property
-    def completed_agreement(self):
-        return bool(self.poc_phone)
+
+class BandPanel(MagModel):
+    band_id = Column(UUID, ForeignKey('band.id'))
+    # This needs to be a nullable integer rather than a nullable boolean to prevent SQLAlchemy from setting a False value
```

Comment:
Just a reminder to myself that this workaround is no longer needed per Eli's previous comment.